### ISSUE TITLE
INC-3  wrong cloud SDK version is deployed on lab.glasswall

### ISSUE THREAD

**Issue by dpatel7** [filetrust/cdr-plugin-folder-to-folder-bugs#51]
# Sample Incidents Playbook

<br>

## Incident Response Team:

**Incident First Responder -** Georgina

**Incident Manager-**

**Incident communicator/reporter-**

**Scope:**

To be used in the event of a infrastructure or service failure or response.

**Incident Response Team:**

***Incident First Responder***

-   Is whoever from the Glasswall information Security and IT Support Squad gets notified or notices the incident

-   Does immediate necessary steps to prevent further breach or exploitation of affected assets (or if necessary delegates this task to someone who can execute these emergency procedures)

-   Initiates Slack channel and mobilises the incident response team with new incident name and the incident response team along with creating a Jira issue

***Incident Manager***

-   Assigns roles within the team and delegates tasks on Jira

-   Asks key questions from the team

-   Closes the incident

-   Incident Handlers

-   Collect evidence

-   Execute various digital forensic and technical tasks required to contain and investigate incident

-   Answer key questions

***Incident communicator/reporter***

-   Communicates with authorities and other third parties that need to be involved

-   Communicates with rest of the Glasswall team and the executive team about the incident

-   Formulates report at the end of the incident based on the evidence collected and formulates future steps and controls in cooperation with the InfoSec squad and CISO

## Tasks:

- [ ] T1. Assign Incident Response Team Roles
- [ ] T2. Start Slack Channel with Incident Name and mobilise team<br>
- [ ] T3. Determine incident severity (see table)
- [ ] T4. Open a Github incident issue<br>
- [ ] T5. Identify immediate steps to prevent further harm to the Glasswall assets<br>
- [ ] T6. Identify steps necessary to regain or ensure business continuity<br>
- [ ] T7. Identify next steps – update policies, processes and identify controls
- [ ] T8. Write Summary to PM and Dinis<br>

## Questions:

- [ ] Q1. Note Version details for Application under investigation<br>
- [ ] Q2. What is the impact?<br>
- [ ] Q3. What caused the outage?<br>
- [ ] Q4. What company asset(s) are involved?<br>
- [ ] Q5. Is business continuity ensured and if not, how do we enable it ASAP?<br>
- [ ] Q6. Do we have a disaster recovery plan and how do we put it into action?<br>
- [ ] Q7. What are the next steps in preparation for future incidents?<br>
- [ ] Q8. Has a bug been reported and is it scheduled for a fix?<br>
- [ ] Q9. What is the incident resolution?<br>

<p><img align="left" width="400" height="400" src="https://user-images.githubusercontent.com/79467690/120673627-a1e04c00-c493-11eb-8b1d-698b449b74b0.png"></p>

**Comment by dpatel7** [comment]
comment from @GINAGC 

Incident reported:
Issue 1: If I go to the https://labs.glasswallsolutions.com:8080/swagger/index.html
Scroll to the bottom select GET - /api/detail/version
I get this result:
access-control-allow-headers: * 
access-control-allow-origin: * 
access-control-expose-headers: * 
content-type: application/json; charset=utf-8 
date: Tue,08 Jun 2021 01:43:05 GMT 
server: Kestrel 
via: proxy-rest-api-69f5cf48f6-grq7m 
x-sdk-api-version: 0.1.17 
x-sdk-engine-version: 1.157
Please note the last 2 lines.
On this site https://github.com/k8-proxy/k8s-compliant-kubernetes/releases/tag/v0.1.2  I see this…
Components part of this release:
·       GW Cloud SDK V0.1.14 https://github.com/k8-proxy/cs-k8s-api/releases/tag/v0.1.14
·       Complaint K8 ICAP https://github.com/k8-proxy/k8s-compliant-kubernetes/releases/tag/v0.1.2 (internal release)
·       CK8s version is v1.18.10
·       Filedrop v.0.1.0 https://github.com/k8-proxy/k8-rebuild-file-drop/releases/tag/v.0.1.0
·       Management UI version is v1.4.8
·       SDK engine 1.157
Issue 2:
From this site I pulled this eu-west-2: ami-07d5b027fb2dddfa9 AMI and I got the results below
x-sdk-api-version: 0.1.11 
x-sdk-engine-version: 1.157
The only constant is the Core SDK version information 1.157

So basically 2 issues here:
- We should not be using 0.1.17 Cloud SDK version on labs, because it is a version under testing. 
- The tagged version as 0.1.14 should display that version.

https://glasswall-solutions.slack.com/archives/C0247M5HJ75/p1623126429010900

**Comment by dpatel7** [comment]
@GINAGC who do you think best for managing this incident?

**Comment by michaelhidalgo** [comment]
Hi @GINAGC, which resource is most appropriate to work on this?

**Comment by samarthdd** [comment]
this is resolved in latest AMIs

<img width="1430" alt="Screenshot 2021-06-21 at 7 08 57 PM" src="https://user-images.githubusercontent.com/64204445/122771797-64b5ef80-d2c4-11eb-8bf0-70b57d45ff73.png">

**Comment by GINAGC** [comment]
This is resolved.
